recipes/kalign2/meta.yaml

```diff
@@ -0,0 +1,24 @@
+package:
+  name: kalign2
+  version: "2.0.3"
+
+build:
+  number: 0
+
+source:
+  fn: kalign-2.0.3.tar.gz
+  url: http://msa.sbc.su.se/downloads/kalign/current.tar.gz
+  md5: bde173eb65e617ed73cd78bad7f881a3
+
+requirements:
+  build:
+    - gcc   # [linux]
```

Comment:
please include `llvm` for osx as described in the guidelines.
https://bioconda.github.io/guidelines.html#c-c